Packs/CommonScripts/Scripts/ParseEmailFiles/ParseEmailFiles.py

```diff
@@ -3719,7 +3746,14 @@ def handle_eml(file_path, b64=False, file_name=None, parse_only_headers=False, m
                 'Text': convert_to_unicode(text, is_msg_header=False),
                 'Headers': header_list,
                 'HeadersMap': headers_map,
-                'Attachments': ','.join(attachment_names) if attachment_names else '',
+                'Attachments': [
+                    {
+                        "Name": attachment_names[i],
+                        "Content-ID": attachment_content_ids[i],
+                        "Content-Disposition": attachment_content_dispositions[i],
+                        "FilePath": attachment_internal_path[i]
+                    } for i in range(len(attachment_names))
+                ],
```

Comment:
this is breaking backward compatibility.
we can add this data under different prop name.
like `AttachmentsData`

Reply:
Hello @anara123 

This is correct. However, is there any other reason we want to keep `Attachments` the way it is? It contains exactly the same infor as `AttachmentNames` which we have already, only the names are not in a list, but join together. This can easily be done using a transformer anyway.

Moreover, `Attachments` wasn't even included in the yml file as an output, so I doubt anyone was ever using it instead of `AttachmentNames`.

Thanks

Reply:
Correct, the reason we return `Attachments` is because of backward compatibility. This is an output from older versions. That is why we don't included it in the yml (in order to prevent from users to use it), but we do return it to the  context because of BC.
So please update your code.